### DISCUSSION TITLE
ShapeFile Harmonization with QGIS

### DISCUSSION THREAD

**Discussion by mayagomez** [cnr-isti-vclab/TagLab#17] [Ideas]
Hi! I have been able to successfully open a ShapeFile created in TagLab in QGIS, however I have not had any luck doing the reverse. Do you hope to add a feature that allows ShapeFiles created elsewhere (ie. QGIS) in TagLab? Thanks!

**Comment by maxcorsini** [comment]
Hi,

it is not possible to import shapefiles in TagLab directly. 

I suggest you to rasterize your shapefiles in a label map (using GIS) and then, import such label map into TagLab. 

I hope this helps.

Cheers
